### DISCUSSION TITLE
页眉内容如何修改

### DISCUSSION THREAD

**Discussion by jiaqiwang969** [sjtug/SJTUThesis#856] [灌水区]
希望将页眉修改为：申请上海交通大学工学博士学位论文

请问如何修改？

**Comment by LogCreative** [comment]
https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L46-L61

将对应的 `\sjtu@name@subject@zh` 改为 `申请上海交通大学工学博士学位论文`。
